bin/cli.ml

```diff
@@ -48,6 +48,12 @@ let silent_eval =
     (fun x -> `Silent_eval x)
     Arg.(value & flag & info [ "silent-eval" ] ~doc)
 
+let record_backtrace =
+  let doc = "Print backtraces when evaluating toplevel phrases." in
+  named
+    (fun x -> `Record_backtrace x)
+    Arg.(value & flag & info [ "record-backtrace" ] ~doc)
```

Comment:
For context see https://github.com/realworldocaml/mdx/pull/206#issuecomment-575582487 and https://github.com/realworldocaml/mdx/pull/206#discussion_r373542330.

If there is another way to repair the broken test, I am happy to extract this into another pull request. But if I revert these changes I still get the error.

Reply:
@Julow I specifically asked for this change. Because one of our new dependencies has the side effect of enabling backtraces, we have to explicitly disable them in the code.

I want this to be configurable, even if the default is to disable backtraces.

Reply:
> I want this to be configurable, even if the default is to disable backtraces.

Why is disabling them in code not enough ?

Reply:
Because in some cases you might want to be able to show the backtraces in a toplevel code fragment.

I think it makes sense to add an escape hatch rather than turning them off with no proper way to re-enable them.

Backtraces being off is a good default, I'm just thinking about the other cases here.

Reply:
I think this shouldn't be included in this PR. My idea is that we should keep the previous behavior (backtraces disabled ?) and open an issue to talk about how to handle them correctly.